python/plugins/processing/algs/gdal/CreateCog.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+
+"""
+***************************************************************************
+    translate.py
+    ---------------------
+    Date                 : August 2012
+    Copyright            : (C) 2012 by Victor Olaya
+    Email                : volayaf at gmail dot com
+***************************************************************************
+*                                                                         *
+*   This program is free software; you can redistribute it and/or modify  *
+*   it under the terms of the GNU General Public License as published by  *
+*   the Free Software Foundation; either version 2 of the License, or     *
+*   (at your option) any later version.                                   *
+*                                                                         *
+***************************************************************************
+"""
+
+__author__ = 'Victor Olaya'
+__date__ = 'August 2012'
+__copyright__ = '(C) 2012, Victor Olaya'
+
```

Comment:
Also need to be updated.

Reply:
Even if it's a copy/paste ? (I update just the dates ?)